src/validation/add.trust.validation.ts

```diff
@@ -0,0 +1,7 @@
+import { body } from "express-validator";
+
+import { ErrorMessages } from "./error.messages";
+
+export const addTrust = [
```

Comment:
Looking at overseas-entities-web/src/validation/fields/nature-of-control.validation.ts I think we should name this variable either addTrustValidations or add_trust_validations to make it clearer that this variable is to do with validatioh